src/client/pages/registration/components/Molecules/RegistrationInputs.jsx

```diff
@@ -0,0 +1,14 @@
+import { FormInput } from '#components/Atoms/FormInput/FormInput.jsx';
+import css from './RegistrationInputs.module.css';
+
+const RegistrationInputs = () => {
+  return (
+    <div className={css.RegistrationInputs}>
+      <FormInput type="name" inputName="name" placeholder="Name" darkOrLight="dark"/>
+      <FormInput type="email" inputName="email" placeholder="Email" darkOrLight="dark"/>
+      <FormInput type="password" inputName="password" placeholder="Password" darkOrLight="dark"/>
```

Comment:
When your components are doubling, or trippling some same properties its good idea while creating template of this component to make this property with some default value.

f.ex. 
TemplateComponent = { defaultProp = value } => ...logic }

Reply:
Did you implement default prop @dag-szad ❓